common/pop_types/cwp_clergy.txt

```diff
@@ -7,7 +7,7 @@
 		111
 	}
 	strata = middle
-	start_quality_of_life = 8
+	start_quality_of_life = 10
```

Comment:
Clergy are meant to have a shittier QoL than someone working as an Academic, Engineer or Professional. Mostly bc being in the clergy doesn't really make you that much money.

Reply:
start_quality_of_life is only used for initialization, nothing else
Same with wealth effect scripts.
This means its effect is forgotten within first months of gameplay.

Reply:
Why, if its only used for initialization, does it need to move up to ten?  What effect are you trying to achieve here?

Reply:
I reverted all changes to start_quality_of_life - I thought it was completely unused